SwiftyJSONAccelerator/SJModelGenerator.swift

```diff
@@ -63,7 +76,7 @@ public class ModelGenerator {
 
      - returns: A ModelGenerator instance.
      */
-    init(baseContent: JSON, prefix: String?, baseClassName: String, authorName: String?, companyName: String?, type: String, filePath: String, includeSwiftyJSON: Bool) {
+    init(baseContent: JSON, prefix: String?, baseClassName: String, authorName: String?, companyName: String?, type: String, filePath: String, supportSwiftyJSON: Bool, includeSwiftyJSON: Bool, supportObjectMapper: Bool, includeObjectMapper: Bool, supportNSCoding: Bool) {
```

Comment:
Probably worth keeping a persistent generator object that the UI updates, rather than creating a new generator each time and needing to pass all the options. It would be cleaner for a lot of things.